pretix_covid_certificates/signals.py

```diff
@@ -139,3 +145,4 @@ def api_event_settings_fields(sender, **kwargs):
 settings_hierarkey.add_default('covid_certificates_record_validity_time', False, bool)
 settings_hierarkey.add_default('covid_certificates_accept_eudgc', True, bool)
 settings_hierarkey.add_default('covid_certificates_accept_manual', True, bool)
+settings_hierarkey.add_default('covid_certificates_combination_rules', '', str)
```

Comment:
```suggestion
settings_hierarkey.add_default('covid_certificates_combination_rules', DEFAULT_COMBINATION_RULES[0], str)
```

(and of course
```
from pretix_covid_certificates.views import DEFAULT_COMBINATION_RULES
```

Without, the API/pretixSCAN will work - but having an obligatory radioselect in the settings without any predefined choice is not that nice...